ecl/ecl-package/CMakeLists.txt

```diff
@@ -56,7 +56,12 @@ include_directories (
 ADD_DEFINITIONS( -D_CONSOLE )
 
 HPCC_ADD_EXECUTABLE ( ecl-packagemap ${SRCS} )
-add_dependencies ( ecl-packagemap espscm ws_packageprocess )
+
+set( WS_PACKAGEPROCESS_DEP )
+if( NOT CLIENTTOOLS_ONLY )
```

Comment:
Remove ws_xxx dependencies from clienttools components.
Tried CMake 2.8.3 with the fix CMP0026 but it doesn't work. 
 I set HPCC Platform and LN minimum CMake version to 2.8.11 in the top level CMakeLists.txt files. I didn't try CMake version between 2.8.3 and 2.8.11.

Will update HPCC github CMake Version if the old version is set.